### PR TITLE
Move index.html to root of generated apps

### PR DESCRIPTION
**Pull request by tpluscode**
## Why?

Currently apps generated from `npm init @open-wc` have all source in the src directory.

This is rather weird when running with dev servers because the URL is similar to `http://localhost:8080/src/` and the `src` segment will be absent from production environment. It also applies to serving prod build with `http-server`.

With `index.html` placed in the root, there is no URL mismatch between environments.

### CODE REVIEW

**Comment by thepassle** [comment]
This also needs to be changed in the `starter-app` generator. Could you change it there, and also do a double check to see if its changed everywhere consistently? :)

**Comment by tpluscode** [comment]
Done @thepassle 

I don't find any more mentions of `index.html`

File: packages/create/src/generators/barebone-app/templates/static/index.html [hunk 1]
```diff
@@ -5,14 +5,14 @@
   <meta name="viewport" content="width=device-width, initial-scale=1.0, viewport-fit=cover" />
   <meta name="description" content="Put your description here.">
 
-  <script type="module" src="./<%= tagName %>.js"></script>
+  <script type="module" src=".src//<%= tagName %>.js"></script>
```
Comment (thepassle) [inline packages/create/src/generators/barebone-app/templates/static/index.html]:
```suggestion
  <script type="module" src="./src/<%= tagName %>.js"></script>
```

**Comment by thepassle** [comment]
There are still quite some mentions of the `src/index.html` in the documentation, could you change those as well to keep everything in sync? :)

**Comment by LarsDenBakker** [comment]
Thanks for this PR!

**Comment by tpluscode** [comment]
Sure, let me check the doc pages

File: packages/building-rollup/README.md [hunk 2]
```diff
@@ -53,15 +53,15 @@ To illustrate this, here is an example of scripts which our rollup config will *
...
+    import { MyApp } from './src/my-app';
   </script>
 
   <!--
     other-script.js is not inline, it is sourced, but as
     it is not type="module", it will still not be picked up
     by rollup and needs to be copied separately
   -->
-  <script src="other-script.js"></script>
+  <script src="src/other-script.js"></script>
```
Comment (thepassle) [inline packages/building-rollup/README.md]:
```suggestion
  <script src="./src/other-script.js"></script>
```

**Comment by thepassle** [comment]
One final comment, otherwise LGTM. @LarsDenBakker @daKmoR can one of you merge?

**Comment by LarsDenBakker** [comment]
@daKmoR how should we split this up in commits?

**Comment by daKmoR** [comment]
no need to split it :)

@tpluscode thaankk you for your contribution ❤️

**Pull request merged**